contentcuration/contentcuration/utils/audit_channel_licenses.py

```diff
@@ -0,0 +1,210 @@
+"""
+Utility functions for auditing channel licenses for community library submission.
+"""
+import logging
+import os
+
+from django.conf import settings
+from django.core.files.storage import default_storage as storage
+from kolibri_content.models import ContentNode as KolibriContentNode
+from kolibri_public.utils.export_channel_to_kolibri_public import (
+    using_temp_migrated_content_database,
+)
+from le_utils.constants import content_kinds
+from le_utils.constants import licenses
+
+from contentcuration.models import AuditedSpecialPermissionsLicense
+from contentcuration.models import Change
+from contentcuration.models import Channel
+from contentcuration.models import License
+from contentcuration.models import User
+from contentcuration.viewsets.sync.constants import CHANNEL
+from contentcuration.viewsets.sync.utils import generate_update_event
+
+logger = logging.getLogger(__name__)
+
+
+def get_content_db_path(channel_id, version=None):
+
+    if version is not None:
+        return os.path.join(settings.DB_ROOT, f"{channel_id}-{version}.sqlite3")
+    return os.path.join(settings.DB_ROOT, f"{channel_id}.sqlite3")
```

Comment:
Apologies, I did not specify this in my comment 😓. I've been thinking about where we should place this function. I think the best module for it to live would be in the `utils/publish.py` module, since at the end, this is the module that creates these content databases, and it would make more sense for other modules to import this function from `utils/publish.py` than from `utils/audit_channel_licenses.py`